recipes/esmp/build.sh

```diff
@@ -19,18 +19,8 @@ else
     export ESMF_OPENMP="ON"
 fi
 
-# OPENMPI bits
-CONDA_LST=`conda list`
-if [[ ${CONDA_LST}'y' == *'openmpi'* ]]; then
-    export CC=mpicc
-    export CXX=mpicxx
-    export LC_RPATH="${PREFIX}/lib"
-    export DYLD_FALLBACK_LIBRARY_PATH=${PREFIX}/lib
-    # ESMF_COMM env variable, choices are openmpi, mpiuni, mpi, mpich2, or mvapich2
-    export ESMF_COMM="openmpi"
-else
-    export ESMF_COMM="mpiuni"
-fi
+# OPENMPI bits would go it
```

Comment:
As soon as this becomes a feedstock we can open an issue to track the development of the "`features`" for the various options.